x-pack/plugins/apm/server/lib/transactions/charts/get_timeseries_data/fetcher.ts

```diff
@@ -48,7 +48,7 @@ export function timeseriesFetcher({
     ...getDocumentTypeFilterForAggregatedTransactions(
       searchAggregatedTransactions
     ),
-    ...uiFiltersES,
+    ...('uiFiltersES' in setup ? setup.uiFiltersES : []),
```

Comment:
Can we default to an empty array in `setupRequest`? That way `uiFiltersES` can be required and we don't have to handle this everywhere.